astropy/nddata/mixins/ndarithmetic.py

```diff
@@ -388,60 +388,40 @@ def _arithmetic_data(self, operation, operand, **kwds):
                         self.data,
                         unit=dimensionless_unscaled,
                         dtype=None,  # use "normal numpy.dtype introspection"
-                        copy=COPY_IF_NEEDED
+                        copy=COPY_IF_NEEDED,
                     ),
                     Quantity(
-                        operand.data,
-                        unit=operand.unit,
-                        dtype=None,
-                        copy=COPY_IF_NEEDED
```

Comment:
(just passing by, but if you prefer this style, you can force it onto ruff by manually adding a [trailing comma](https://docs.astral.sh/ruff/formatter/#line-breaks))